lib/web/apiserver.go

```diff
@@ -989,7 +989,11 @@ func (h *Handler) bindDefaultEndpoints() {
 	// User Status (used by client to check if user session is valid)
 	h.GET("/webapi/user/status", h.WithAuth(h.getUserStatus))
 
+	// TODO(kimlisa): DELETE IN 20.0 along with the api path defined in `config.ts`
+	// Replaced by its v2 endpoint
 	h.GET("/webapi/roles", h.WithAuth(h.listRolesHandle))
+	// v2 introduces a query param for optionally including system roles in the list.
+	h.GET("/v2/webapi/roles", h.WithAuth(h.listRolesHandle))
```

Comment:
Does this need a `v2` version bump? 

We're only adding an extra parameter - does it break regular v1 behavior or need to?

Reply:
it'll kinda break the feature i'm working on that relies on this endpoint to return with system roles. without a v2 endpoint, a request could go to an older proxy and return empty results. i wouldn't know if it was empty because there were no roles found, or it went to an older proxy that didn't process the new query param (default was to return without system roles)